src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1011,6 +1012,34 @@ def load_arguments(self, _):
     with self.argument_context('aks mesh upgrade start') as c:
         c.argument('revision', validator=validate_azure_service_mesh_revision, required=True)
 
+    with self.argument_context('aks approuting enable') as c:
+        c.argument('enable_kv', options_list=[
+                   '--enable-kv'], required=False, action='store_true')
```

Comment:
`required=False` is the default setting and does not require explicit declaration
```suggestion
        c.argument('enable_kv', action='store_true')
```